main.yml

```diff
@@ -696,14 +699,22 @@ streams:
       - id: kafkas
         title: "Kafka Instances"
         default: true
-      - id: service-accounts
-        title: "Service Accounts"
       - id: resources
         title: "Resources"
       - id: streams-docs
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
 
+service-registry:
+  title: "Service Registry"
+  frontend:
+    sub_apps:
+      - id: sr
+        title: "Service Registry"
```

Comment:
```suggestion
        title: "Instances"
```
Title  should be `Instances` according to  design doc.  https://marvelapp.com/prototype/5dhc6c0/screen/81843300
@jgiardino ^^

Reply:
This is also inconsistent with Kafka...

Reply:
> This is also inconsistent with Kafka...

I think we should have `Instances` for both kafka ad SR

Reply:
Tiffany updated the design to "Service Registry Instances"

Reply:
**https://prod.foo.redhat.com:1337/beta/application-services/overview**
![Screenshot from 2021-09-16 19-06-49](https://user-images.githubusercontent.com/19213040/133625318-caa53de1-a290-4b7a-bb27-6533e9b660dc.png)

**https://prod.foo.redhat.com:1337/beta/application-services/service-accounts**
![Screenshot from 2021-09-16 19-07-28](https://user-images.githubusercontent.com/19213040/133625336-653287be-84b5-41b3-8a57-9b8696819e79.png)

**https://prod.foo.redhat.com:1337/beta/application-services/service-registry**
![Screenshot from 2021-09-16 19-07-53](https://user-images.githubusercontent.com/19213040/133625345-89c762bf-acce-49f1-85d1-fd8a5ffa200c.png)


@pmuir @ajaypratap003 This is how the UI looks now, after the requested changes. Please let me know if there is any other change required.
Still not sure how everything integrates together!!